recipes/qemu/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+mkdir build
+cd build
+
+# bfd ld doesn't like the deprecated linker options that
+# qemu 5.0.0 uses.  Link with GOLD instead
+export LD="$LD_GOLD"
+
+#host_cc defaults to hardcoded 'cc' and we're not cross-compiling
+#for real so set it to the long name
+../configure --prefix="$PREFIX" \
+             --host-cc="$CC" \
```

Comment:
Probably also need to set ``--interp-prefix`` because I think it is still defaulting to something in `/usr`.  It is the path where at runtime, qemu will look for shared libraries for the guest.  Probably needs to be set to something like `$PREFIX/gnemul/qemu-%M`.  I'm not sure exactly where this is used.  It's probably for partial emulation of some kind that I don't use very often.

Reply:
It is the default value for the runtime `-L` option also set from env var `QEMU_LD_PREFIX`.  Since it can be overridden by the cmdline or env at runtime, it isn't critical to set but perhaps it should still be and other packages could install stuff into that prefix to bundle up a usable qemu system.